assets/js/googlesitekit/widgets/components/WidgetAreaHeader.js

```diff
@@ -80,7 +80,15 @@ export default function WidgetAreaHeader( {
 WidgetAreaHeader.propTypes = {
 	slug: PropTypes.string.isRequired,
 	Icon: PropTypes.bool,
-	title: PropTypes.string,
-	subtitle: PropTypes.oneOf( PropTypes.string, PropTypes.elementType ),
-	CTA: PropTypes.object,
+	title: PropTypes.oneOfType( [ PropTypes.string, PropTypes.object ] ),
```

Comment:
We render the title as is (`{ title }`), then it can be either a string or an already rendered element:

```suggestion
	title: PropTypes.oneOfType( [ PropTypes.string, PropTypes.element ] ),
```